tests/layer_tests/tensorflow_tests/test_tf_SegmentSum.py

```diff
@@ -69,3 +69,65 @@ def test_segment_sum_different_types(self, params, ie_device, precision, ir_vers
         self._test(*self.create_segment_sum_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+class TestSegmentSum(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'data:0' in inputs_info, "Test error: inputs_info must contain `data`"
+        assert 'segment_ids:0' in inputs_info, "Test error: inputs_info must contain `segment_ids`"
+        data_shape = inputs_info['data:0']
+        segment_ids_shape = inputs_info['segment_ids:0']
+        inputs_data = {}
+        inputs_data['data:0'] = np.random.randint(-50, 50, data_shape)
+        # segment_ids data must be sorted according to TensorFlow SegmentSum specification
+        inputs_data['segment_ids:0'] = np.sort(np.random.randint(0, 20, segment_ids_shape))
+        return inputs_data
+
+    def create_segment_sum_net(self, data_shape, segment_ids_shape, data_type, segment_ids_type):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            data = tf.compat.v1.placeholder(data_type, data_shape, 'data')
```

Comment:
please check other test with complex type. OV does not handle Placeholder of complex type.